### PR TITLE
Homepage [Hero Section, Booking Section, Plan Your trip]

### PR DESCRIPTION
**Pull request by Pheakdey666**
Please Review My Code !

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,10 +4,212 @@
...
+            </a>
+          </div>
+        </div>
+        <div class="main-img">
+          <img class="main-car-img" src="/images/cars/main-car.png" alt="" />
+        </div>
+      </div>
+    </section>
+
+    <!-- Book Car Section -->
```
Comment (miratorimoonlight) [inline index.html:86]: Book car section cannot be selected.

File: index.html [hunk 2]
```diff
@@ -4,10 +4,212 @@
...
+        <div class="form">
+          <div class="form-box-1">
+            <div class="select-car">
+              <label for="car">
+                <i class="fa-solid fa-car"></i>
+                &nbsp; Select your Car Type<b>*</b></label
+              >
+              <select name="" id="">
+                <option>Select your car type</option>
+                <option value="Camry">Camrry</option>
```
Comment (miratorimoonlight) [inline index.html:100]: Please add more options than camry

File: index.html [hunk 3]
```diff
@@ -4,10 +4,212 @@
...
+            Rent the car of your dreams. Unbeatable prices, unlimited miles,
+            flexible pick-up options and much more.
+          </p>
+          <div class="content-btn">
+            <a class="content-btn-book-ride" href="#"
+              >Book Ride &nbsp;
+              <i class="fa-solid fa-circle-check"></i>
+            </a>
+            <a class="content-btn-learn-more" href="#"
+              >Learn More &nbsp;
```
Comment (miratorimoonlight) [inline index.html:75]: link to about page

**Review by miratorimoonlight — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Pheakdey666) [reply index.html:86]: Already Work!

**Review by miratorimoonlight — APPROVED** [review]

**Pull request merged**